doc/src/fpga_api/fpga_api.rst

```diff
@@ -1,4 +1,6 @@
-# OPAE C API Reference
+====================
+OPAE C API Reference
+====================
```

Comment:
These may clash with other documentation work - can you make sure those are up to date, too?

Reply:
ok yes